setup.py

```diff
@@ -1,3 +0,0 @@
-from setuptools import setup
-
-setup()
```

Comment:
@inmzhang Do you happen to know why we have a `setup.py` file in the repo? Since there is a `toml` file, I think we no longer need `setup.py`. 

https://pip.pypa.io/en/latest/reference/build-system/setup-py/#editable-installation